### PR TITLE
Feature/#24 mobile device support

### PR DESCRIPTION
**Pull request by stianlaa**
I was looking around for something to use "out of the box", to get gestures such as pinch and pan such as ZingTouch, or HammerJs (which I discovered was deprecated), ended up doing it manually, and it worked out okay.

This PR contains code to support drawing with hands/touch devices, as well as support for the pinch-gesture for zooming using two fingers.

I'll probably do a quick rework of the setup so that touchCache contains the previous events, I think that would give a more natural feel to the gestures.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,40 +1,42 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es2017", "es7", "es6", "dom"],                             /* Specify library files to be included in the compilation. */
+    "target": "es6", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```
Comment (holwech) [inline tsconfig.json]: Can you revert the changes made in this file?

**Review by holwech — CHANGES_REQUESTED** [review]

**Comment by holwech** [comment]
I'm getting the following error in Firefox:

```
Uncaught ReferenceError: TouchEvent is not defined
    eventToPointerEvent Dispatcher.js:125
    dispatchEvent Dispatcher.js:81
    onMouseLeave MouseEventListenerController.js:53
    onMouseDown MouseEventListenerController.js:33
```

**Comment by holwech** [comment]
Overall this looks good. Some things to think about (for the future):

1. Does having touch and pointer events running in parallel cause reduction in performance?
2. Touch and pointer event listener classes are quite similar. Does it make sense to have them in separate classes?

I don't have any answers here, but had this been our full-time job it would maybe be worth having a look into this. Since we have limited resources on this, I don't know it's worth it...

File: tsconfig.json (hunk 1 above)
Comment (stianlaa) [reply tsconfig.json]: Absolutely, I see the format on save has been causing trouble.

**Comment by stianlaa** [comment]
Will re-open the pull-request when I have corrected according to your comments, and fixed panning/zoom with delta
